### PR TITLE
#510 Add Page type to PageChooser button label

### PR DESCRIPTION
**Pull request by cho-leukeleu**
This should fix https://github.com/wagtail/wagtail/issues/510

As the original issue did not take into account that it is possible to specify more than one page type I chose to put all page types in parenthesis (which can actually lead to a pretty large button I guess :/).

### CODE REVIEW

File: wagtail/wagtailadmin/widgets.py [hunk 1]
```diff
@@ -137,6 +137,8 @@ class AdminPageChooser(AdminChooser):
     def __init__(self, target_models=None, can_choose_root=False, **kwargs):
         super(AdminPageChooser, self).__init__(**kwargs)
 
+        if target_models:
+            self.choose_one_text += ' (' + ', '.join([model._meta.model_name for model in target_models]) + ')'
```
Comment (BertrandBordage) [inline wagtail/wagtailadmin/widgets.py]: Could you use `verbose_name` instead of `model_name`, please? :)
I think `model_name` is not localized.

Comment (cho-leukeleu) [inline wagtail/wagtailadmin/widgets.py]: Done :). Thanks for reviewing!

File: wagtail/wagtailadmin/widgets.py [hunk 2]
```diff
@@ -137,6 +137,9 @@ class AdminPageChooser(AdminChooser):
     def __init__(self, target_models=None, can_choose_root=False, **kwargs):
         super(AdminPageChooser, self).__init__(**kwargs)
 
+        if target_models:
+            # import pdb; pdb.set_trace()
```
Comment (robmoorman) [inline wagtail/wagtailadmin/widgets.py]: Please remove this comment

**Comment by BertrandBordage** [comment]
So in fact target_model is never `None`? Maybe it should become a mandatory argument, then.

**Comment by gasman** [comment]
@BertrandBordage For a `PageChooserPanel` it's never `None`, because it picks up the page type from the foreign key - however, there are other places where `AdminPageChooser` is used (such as `PageChooserBlock` in a StreamField) and for those, `target_model` might be omitted.

**Comment by BertrandBordage** [comment]
Ah, ok, thanks for explaining.

**Comment by gasman** [comment]
Merged in 67ff4c26df162a3a7de3d46a5f826874e76ff8e5 (with a small tweak to ensure that the `Page` special case still works when other translations are active) - thanks @cho-leukeleu!

**Pull request closed without merging**